--- init.lua.orig	2016-08-05 20:39:45.000000000 -0700
+++ init.lua	2016-08-12 10:13:45.062742423 -0700
@@ -200,9 +200,11 @@
 	collisionbox = selectionbox_bear
 	})
 
-if factions~= nil and
-	type(factions.set_base_reputation) == "function" then
-	factions.set_base_reputation("bears","players",-25)
+local FACTIONSDIR = minetest.get_modpath("factions")
+if FACTIONSDIR then
+	if factions~= nil and type(factions.set_base_reputation) == "function" then
+		factions.set_base_reputation("bears","players",-25)
+	end
 end
 
 minetest.log("action","\tadding mob "..bear_prototype.name)
