--- builtin.lua.orig	2016-02-15 05:19:00.000000000 -0800
+++ builtin.lua	2016-08-12 09:59:33.153573245 -0700
@@ -38,7 +38,7 @@
 
 -- check if some settings are invalid
 local enable_hunger = minetest.setting_getbool("hud_hunger_enable")
-if (enable_hunger == true or HUD_ENABLE_HUNGER == true) and not hud.show_hunger then
+if enable_hunger == true and not hud.show_hunger then
 	hud.notify_hunger(5)
 end
 
