--- recipes.lua.orig	2016-05-12 14:59:16.000000000 -0700
+++ recipes.lua	2016-08-12 11:19:15.783087077 -0700
@@ -51,13 +51,15 @@
 	end
 	
 	technic.recipes[typename].recipes[index] = recipe
-	if unified_inventory and technic.recipes[typename].output_size == 1 then
-		unified_inventory.register_craft({
-			type = typename,
-			output = data.output,
-			items = data.input,
-			width = 0,
-		})
+	if minetest.get_modpath("unified_inventory") then
+		if unified_inventory and technic.recipes[typename].output_size == 1 then
+			unified_inventory.register_craft({
+				type = typename,
+				output = data.output,
+				items = data.input,
+				width = 0,
+			})
+		end
 	end
 end
 
