--- init.lua.orig	2016-07-29 14:48:42.000000000 -0700
+++ init.lua	2016-08-12 09:29:54.842826179 -0700
@@ -573,8 +573,8 @@
 	tool_capabilities = {
 		max_drop_level=0,
 		groupcaps={
-			choppy={times={[2]=7.50, [3]=2.80}, maxwear=0.01, maxlevel=1},
-			fleshy={times={[2]=5.50, [3]=2.80}, maxwear=0.01, maxlevel=1}
+			choppy={times={[2]=7.50, [3]=2.80}, uses="maxwear", maxlevel=1},
+			fleshy={times={[2]=5.50, [3]=2.80}, uses="maxwear", maxlevel=1}
 		}
 	},
 })
